src/fastcs/controller.py

```diff
@@ -19,9 +20,14 @@ class SingleMapping:
 
 
 class BaseController:
+    #! Attributes passed from the device at runtime.
```

Comment:
Whoops, I meant to do [this](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#doc-comments-and-docstrings) but used `!` instead of `:`.

Reply:
Oh, I have never seen that. I have always used `"""` on the line below for docstrings.